src/main/views/features/limited/company-inactive/company-inactive.njk

```diff
@@ -1,17 +1,17 @@
 {% extends "layouts/default.njk" %}
-
 {% block main_content %}
-  <main class="govuk-main-wrapper" id="main-content" role="main">          
+  <div class="govuk-main-wrapper" id="main-content" role="main">
```

Comment:
the attributes class="govuk-main-wrapper", id="main-content" and  role="main" will need removing from each view as this will also cause accessibility issues with duplicate main landmarks. As they are defined in the default.njk file